scripts/build_and_push_docker_images.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+function buildAndPushDockerImages {
+  cd $1
+  docker build -t registry.ng.bluemix.net/$1/microservice-speaker .
```

Comment:
Why all the images is tagged with microservice-speaker? shouldn't it be `docker build -t registry.ng.bluemix.net/<namespace>/$1`?